doc/whatsnew/fragments/8718.other

```diff
@@ -0,0 +1,3 @@
+Pylint now support python 3.12.
```

Comment:
Do you want to also add the classifier in this PR, or is that done later?

```suggestion
Pylint now supports python 3.12.
```

Reply:
It's also my guess that 3.0 wouldn't be released anyway until 3.12 is in final. We're not even in beta yet, but we should be around the time 3.12 is in final.

Reply:
There's new construct in 3.12, the [type parameter syntax](https://docs.python.org/3.12/whatsnew/3.12.html#pep-695-type-parameter-syntax), we don't handle this yet afaik so we'll have to wait a little before that (like for trystar in 3.11).

Reply:
I will remove this fragment :)